tests/test_parmparse.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import os
+
+import amrex.space3d as amr
+
+
+def test_parmparse():
+    dopml = False
+    ncell = 10
+    dt = 0.0
+    pp = amr.ParmParse("")
+    dir_name = os.path.dirname(__file__)
+    pp.addfile(dir_name + "/parmparse_inputs")
```

Comment:
Slightly more portable / generic (e.g., Unix <-> Windows):
```suggestion
    pp.addfile(os.path.join(dir_name, "/parmparse_inputs"))
```